pkg/ddc/jindo/const.go

```diff
@@ -26,6 +26,9 @@ const (
 	JINDO_MASTER_PORT_MAX   = 11000
 	DEFAULT_WORKER_RPC_PORT = 6101
 	JINDO_WORKER_PORT_MAX   = 8100
+	PORT_NUM                = 2
```

Comment:
camel-case
[CodeReviewComments](https://github.com/golang/go/wiki/CodeReviewComments#initialisms)

Reply:
thx, but the consts seem not to be camel-case under project

Reply:
step-by-step modifications that already exist
but make sure the increments don't increase

Reply:
thx，I'll do it all together next time if necessary